libraries/16.04

```diff
@@ -0,0 +1,513 @@
+libQt5Sql.so.5.5
+libQt5Sql.so.5.5.1
+libapt-inst.so.2.0
+libapt-inst.so.2.0.0
+libapt-pkg.so.5.0
+libapt-pkg.so.5.0.0
+libasprintf.so.0
+libasprintf.so.0.0.0
+libboost_program_options.so.1.58.0
+libcap-ng.so.0
+libcap-ng.so.0.0.0
+libdb-5.3.so
+libdbus-glib-1.so.2
+libdbus-glib-1.so.2.3.3
+libdebconfclient.so.0
+libdebconfclient.so.0.0.0
+libedit.so.2
+libedit.so.2.0.53
+libefiboot.so.0
+libefiboot.so.0.21
+libefivar.so.0
+libefivar.so.0.21
+libestr.so.0
+libestr.so.0.0.0
+libexpatw.so.1
+libexpatw.so.1.6.0
+libffi.so.6
+libffi.so.6.0.4
+libform.so.5
+libform.so.5.9
+libformw.so.5
+libformw.so.5.9
+libfreetype.so.6
+libfreetype.so.6.11.1
+libfwup.so.0
+libfwup.so.0.5
+libgdbm.so.3
+libgdbm.so.3.0.0
+libgdbm_compat.so.3
+libgdbm_compat.so.3.0.0
+libgflags.so.2
+libgflags.so.2.1.2
+libgflags_nothreads.so.2
+libgflags_nothreads.so.2.1.2
+libgio-2.0.so.0
+libgio-2.0.so.0.4705.0
+libgirepository-1.0.so.1
+libgirepository-1.0.so.1.0.0
+libglog.so.0
+libglog.so.0.0.0
+libgmodule-2.0.so.0
+libgmodule-2.0.so.0.4705.0
+libgmp.so.10
+libgmp.so.10.3.0
+libgnutls-deb0.so.28
+libgnutls-deb0.so.28.41.12
+libgnutls-openssl.so.27
+libgnutls-openssl.so.27.0.2
+libgobject-2.0.so.0
+libgobject-2.0.so.0.4705.0
+libgssapi_krb5.so.2
+libgssapi_krb5.so.2.2
+libgthread-2.0.so.0
+libgthread-2.0.so.0.4705.0
+libhogweed.so.4
+libhogweed.so.4.1
+libicudata.so.55
+libicudata.so.55.1
+libicui18n.so.55
+libicui18n.so.55.1
+libicuio.so.55
+libicuio.so.55.1
+libicule.so.55
+libicule.so.55.1
+libiculx.so.55
+libiculx.so.55.1
+libicutest.so.55
+libicutest.so.55.1
+libicutu.so.55
+libicutu.so.55.1
+libicuuc.so.55
+libicuuc.so.55.1
+libidn.so.11
+libidn.so.11.6.15
+libk5crypto.so.3
+libk5crypto.so.3.1
+libkrb5.so.3
+libkrb5.so.3.3
+libkrb5support.so.0
+libkrb5support.so.0.1
+liblockfile.so.1
+liblockfile.so.1.0
+libmenu.so.5
+libmenu.so.5.9
+libmenuw.so.5
+libmenuw.so.5.9
+libmpdec.so.2
+libmpdec.so.2.4.1
+libnetfilter_conntrack.so.3
+libnetfilter_conntrack.so.3.5.0
+libnettle.so.6
+libnettle.so.6.1
+libnfnetlink.so.0
+libnfnetlink.so.0.2.0
+libp11-kit.so.0
+libp11-kit.so.0.1.0
+libpanel.so.5
+libpanel.so.5.9
+libpanelw.so.5
+libpanelw.so.5.9
+libpcap.so.0.8
+libpcap.so.1.7.4
+libpcre16.so.3
+libpcre16.so.3.13.2
+libpcreposix.so.3
+libpcreposix.so.3.13.2
+libpng12.so.0
+libproxy.so.1
+libproxy.so.1.0.0
+libsemanage.so.1
+libsqlite3.so.0
+libsqlite3.so.0.8.6
+libstdc++.so.6
+libstdc++.so.6.0.21
+libtasn1.so.6
+libtasn1.so.6.5.1
+libtic.so.5
+libtic.so.5.9
+libtpm_unseal.so.1
+libtpm_unseal.so.1.0.0
+libtspi.so.1
+libtspi.so.1.2.0
+libubuntu-download-manager-common.so.1
+libubuntu-download-manager-common.so.1.2.0
+libudm-common.so.1
+libudm-common.so.1.2.0
+libudm-priv-common.so.1
+libudm-priv-common.so.1.2.0
+libunwind-coredump.so.0
+libunwind-coredump.so.0.0.0
+libunwind-ptrace.so.0
+libunwind-ptrace.so.0.0.0
+libunwind-x86_64.so.8
+libunwind-x86_64.so.8.0.1
+libunwind.so.8
+libunwind.so.8.0.1
+libustr-1.0.so.1
+libustr-1.0.so.1.0.4
+libyaml-0.so.2
+libyaml-0.so.2.0.4
```

Comment:
@elopio and @kyrofa done https://github.com/ubuntu-core/snapcraft/pull/335/files#diff-2cb561c5e46512465f54a0f84d421c80R66

But is also makes me think if it is worth to make this dynamic.